app/controllers/coronavirus_local_restrictions_controller.rb

```diff
@@ -36,4 +40,14 @@ def content_item
       end
     end
   end
+
+  def restiction_expiry(search)
```

Comment:
The Cache time stuff looks good. There's a teeny tiny typo

Reply:
Oops - thanks for catching.